single-solution/content/volumetric.php

```diff
@@ -1,15 +1,18 @@
-    <div class="inline-div">
+    
+
+<div class="inline-div">
 		<input id = "solvent_formula" type="text" placeholder="Solvent Chemical Identity"> <br>
 		<input id = "solute_formula" type="text" placeholder="Solute Identity"><br>
 		<input id = "solute_molec_weight" type="text" placeholder="Solute Molecular Weight"><br>
-		<input id= "density" type="text" placeholder="Density of Pure Solute"><br>
+		<input id = "density" type="text" placeholder="Density of Pure Solute"><br>
 		<input id = "solution_concentration" type="text" placeholder="Solute Concentration"><br>
+    <input id= "total_volume" type="text" placeholder="Solution Total Volume (milli Liters)"><br>
```

Comment:
total_volume input from user is required to create a solution. This is one of the major problems gravimetric and volumetric solutions were not working properly.